lizmap/modules/lizmap/lib/Request/Proxy.php

```diff
@@ -21,6 +21,94 @@ class Proxy
      */
     protected static $_profiles = array();
 
+    protected static $services = null;
+
+    protected static $appContext = null;
+
+    public static function setServices($services = null)
```

Comment:
The function name is not related on how the method is used, as it is used to get services.
I think we should have a `getServices()`, which return the current `self::$services` if not null,else `\lizmap::getServices()`. And we should have a `setServices($services)` to set  `self::$services`.